spec/unit/inventory_v4_spec.rb

```diff
@@ -0,0 +1,47 @@
+require_relative '../spec_helper'
+
+module Hawkular::Inventory::RSpecUnit
+  include Hawkular::InventoryV4
+  context 'Inventory v4 unit test' do
```

Comment:
If this is supposed to be added to other test files, then it should be using [shared examples](https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples). If that's not the case, a block describing tests should be `describe`, not `context`. And no need to name it `X unit test`.